todo_app/templates/index.html

```diff
@@ -11,8 +11,19 @@ <h1 class="display-4">To-Do App</h1>
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        {% for item in items %}
+          <li>
+            {{ item.id }} - {{ item.title }} - {{ item.status }} 
+           </li>         
+        {% endfor %}
       </ul>
+      <form method="POST" action="/add">
+        <ul>
+          <li>Title: <input name="title"></li>
+          <li>Status: <input name="status"></li>
```

Comment:
Nice. I would recommend including the `type` of the input, i.e put the attribute `type="text"` along side the `name="..."`.

"text" is the default type but it's nice to be explicit.

Other attributes you could add:
- `required` would make it so that the user cannot submit the form with the input still empty.
- You cold add `value="Not Started"` to the status input to set an initial, default value